common/remote/hooks/azure/azurefile.cpp

```diff
@@ -935,7 +935,7 @@ class AzureFileClient : public AzureAPICopyClientBase
 
     virtual void doStartCopy(const char * source) override
     {
-        fileCopyOp = std::move(fileClient->StartCopy(source));
```

Comment:
These were generating a warning "[Moving a temporary object prevents copy elision -Wpessimizing-move"

As far as I can tell this indicates that the std::move is redundant.

Reply:
I think you're right - not needed on any assignment of a function return